### PR TITLE
modify home screen logged in and logged out

### PR DESCRIPTION
**Pull request by tiffanyqi**
- Add scrollable information for logged out
- Move categories list and tags into a righthand column
- Add options for buttons rather than have it float there

### CODE REVIEW

File: cal/cal/static/css/calendaranalytics.css [hunk 1]
```diff
@@ -10,22 +10,41 @@ h1, h2, h3, h4, h5, h6, a, p {
   color: #000; }
 
 .jumbotron h1, .jumbotron .h1 {
-  color: #000; }
+  color: #000;
```
Comment (MaxFangX) [inline cal/cal/static/css/calendaranalytics.css:13]: These changes should be made in the .scss file. Remember that .css is the compiled code, and will be erased as soon as the sass is recompiled.

Relevant command in cal/README.md

File: cal/cal/templates/home_logged_in.html [hunk 2]
```diff
@@ -41,55 +42,59 @@
...
+        aria-haspopup="true"
+        aria-expanded="false">
+        Select Calendars
+        <span class="caret"></span>
+      </button>
+      <ul class="dropdown-menu">
+        <li ng-repeat="cal in calendars">
+          <a href="#">
+            <input type="checkbox" ng-model="cal.enabled" ng-click="$ctrl.refresh('panalytics')">
+            {% verbatim %} {{ cal.calendar_id }} {% endverbatim %}
```
Comment (MaxFangX) [inline cal/cal/templates/home_logged_in.html:61]: Now that the summary code is merged in, you can do cal.summary here and reference your issue (:

File: .gitignore [hunk 3]
```diff
@@ -11,5 +11,14 @@ cal/cal/static/bower_components/*
...
 
-# Hide environment variables 
+# Hide environment variables
 .ebextensions/02-cal.config
+
+# DS Store
+.DS_Store
+
+# Fishrc
+cal/.fishrc
```
Comment (MaxFangX) [inline .gitignore]: Why is your .fishrc in the ./cal folder, instead of ~/.fishrc ?

**Comment by MaxFangX** [comment]
Made some comments - great work! Make sure you merge master into this branch before you start resolving the comments. It looks like there are some conflicts - feel free to reach out to me if you have trouble

File: cal/cal/templates/home_logged_in.html (hunk 2 above)
Comment (tiffanyqi) [reply cal/cal/templates/home_logged_in.html:61]: @MaxFangX can you clarify what you mean by this? Thanks!

File: cal/cal/static/css/calendaranalytics.css (hunk 1 above)
Comment (tiffanyqi) [reply cal/cal/static/css/calendaranalytics.css:13]: Copied and pasted this code into the scss, thanks for that. What command should I reference in cal/README.md?

Comment (MaxFangX) [reply cal/cal/static/css/calendaranalytics.css:13]: Command: sass --watch .
1) In our project root folder, run the command. This sets sass to watch for any changes in .scss files
2) modify the scss files you copied stuff over into, and save. This is to trigger a recompiling of the css files.

Let me know if you need more help. Here's some sass docs too: http://sass-lang.com/guide

File: cal/cal/templates/home_logged_in.html (hunk 2 above)
Comment (MaxFangX) [reply cal/cal/templates/home_logged_in.html:61]: - The issue I am talking about is https://github.com/MaxFangX/calendar-analytics/issues/19 
- You can reference an issue by making a comment in this pull request.

Comment (tiffanyqi) [reply cal/cal/templates/home_logged_in.html:61]: Ahhh I thought you were talking about modifying the "summary" or some field on the intro page. This makes sense!

**Comment by tiffanyqi** [comment]
@MaxFangX I fixed the above changes. Is it good to merge?

**Pull request merged**